x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/roc_curve_help_popover.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  HelpPopover,
+  HelpPopoverButton,
+} from '../../../../../components/help_popover/help_popover';
+
+export const RocCurveHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton
+          onClick={() => {
+            setIsPopoverOpen(!isPopoverOpen);
+          }}
+        />
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      isOpen={isPopoverOpen}
+      title={i18n.translate('xpack.ml.dataframe.analytics.rocCurvePopoverTitle', {
+        defaultMessage: 'Receiver operating characteristic (ROC) curve',
+      })}
+    >
+      <p>
+        <FormattedMessage
+          id="xpack.ml.dataframe.analytics.rocCurveBasicExplanation"
+          defaultMessage="The receiver operating characteristic (ROC) curve is a plot that represents the performance of the classification process at different predicted probability thresholds."
```

Comment:
I think since "ROC" is already defined twice (in the label and in the popover title), we don't need to define it again. i.e. 
```suggestion
          defaultMessage="The ROC curve is a plot that represents the performance of the classification process at different predicted probability thresholds."
```